ur_robot_driver/test/robot_driver.py

```diff
@@ -223,9 +112,13 @@ def test_start_scaled_jtc_controller(self):
         req = SwitchController.Request()
         req.strictness = SwitchController.Request.BEST_EFFORT
         req.activate_controllers = ["scaled_joint_trajectory_controller"]
-        result = self.call_service("/controller_manager/switch_controller", req)
 
-        self.assertEqual(result.ok, True)
+        self.assertTrue(
+            self._controller_manager_interface.switch_controller(
+                strictness=SwitchController.Request.BEST_EFFORT,
+                activate_controllers=["scaled_joint_trajectory_controller"],
```

Comment:
This seems to be a duplication of the two lines above the assert statement. Probably those above should get deleted?

Reply:
Yes, that was an oversight on my part. Should be fixed now.